### ISSUE TITLE
Fails if app is iOS-only

### ISSUE THREAD

**Issue by iotashan** [junedomingo/react-native-rename#211]
**Describe the bug**
Running npx react-native-rename@latest returns the error `Directory should be created using "react-native init".`

**To Reproduce**
1. run react-native init
2. delete the android directory
3. run react-native-rename

**Expected behavior**
I would expect it to ignore android renames if the android directory is missing, same with iOS. At the very least it should ignore android if I use --iosBundleID instead of -b

**Comment by caio-passos** [comment]
Yep,  when re-naming the repo again the command fails with Directory should be created using "react-native init". It happens with Android while using Expo53
